src/net/Mybuild

```diff
@@ -1,25 +1,27 @@
 package embox.net
 
 module dev {
-	option number netdev_quantity=8
```

Comment:
Sure, I can rollback it.
I just sorted it in alphabetical order.
Do you have any code style for ordering code lines in Mybuild files?